### PR TITLE
Minor code cleanup

### PR DESCRIPTION
**Pull request by cna0**
Replace almost all -include_lib(../../app...) with include_lib(app/...)

Replace almost all -include(../file) with -include(file)

Remove unused bindings

Remove aehttp_api_validate:to_method/1

---

This PR is supported by the Aeternity Crypto Foundation.

### CODE REVIEW

File: apps/aecontract/test/aect_test_utils.erl [hunk 1]
```diff
@@ -62,9 +62,9 @@
 -include_lib("common_test/include/ct.hrl").
 -include_lib("aecontract/include/aecontract.hrl").
 -include_lib("aecontract/include/hard_forks.hrl").
--include_lib("aecontract/test/include/aect_sophia_vsn.hrl").
 -include_lib("aecontract/test/include/aect_contract_cache.hrl").
--include("../src/aect_sophia.hrl").
+-include("../../aecontract/test/include/aect_sophia_vsn.hrl").
```
Comment (uwiger) [inline apps/aecontract/test/aect_test_utils.erl:66]: Isn't this a change in the wrong direction?

File: apps/aehttp/src/aehttp_api_validate.erl [hunk 2]
```diff
@@ -62,8 +62,7 @@ response(OperationId, Method, Code, Response, Validator, EndpointsMod) ->
 
 response_(_OperationId, _Method0, Code, _Response, _Validator, _EndpointsMod) when Code >= 500 andalso Code < 600 ->
     ok;
-response_(OperationId, Method0, Code, Response, Validator, EndpointsMod) ->
-    Method = to_method(Method0),
+response_(OperationId, _Method0, Code, Response, Validator, EndpointsMod) ->
```
Comment (uwiger) [inline apps/aehttp/src/aehttp_api_validate.erl:65]:
```suggestion
response_(OperationId, _Method, Code, Response, Validator, EndpointsMod) ->
```

File: apps/aehttp/src/aehttp_api_validate.erl [hunk 3]
```diff
@@ -107,8 +106,7 @@ fix_def_refs(_, X) ->
     {ok, Model :: #{}, cowboy_req:req()}
     | {error, Reason :: any(), cowboy_req:req()}.
 
-request(OperationId, Method0, Req, Validator, EndpointsMod) ->
-    Method = to_method(Method0),
+request(OperationId, _Method0, Req, Validator, EndpointsMod) ->
```
Comment (uwiger) [inline apps/aehttp/src/aehttp_api_validate.erl:109]:
```suggestion
request(OperationId, _Method, Req, Validator, EndpointsMod) ->
```

File: apps/aecontract/test/aect_test_utils.erl (hunk 1 above)
Comment (cna0) [reply apps/aecontract/test/aect_test_utils.erl]: You're right. Reverted; that one was a mistake.

**Review by uwiger — APPROVED** [review]

**Review by velzevur — APPROVED** [review]

**Pull request merged**